editor/editor_node.cpp

```diff
@@ -7035,7 +7035,7 @@ EditorNode::EditorNode() {
 	ED_SHORTCUT_OVERRIDE("editor/take_screenshot", "macos", KeyModifierMask::META | Key::F12);
 	settings_menu->add_shortcut(ED_GET_SHORTCUT("editor/take_screenshot"), EDITOR_SCREENSHOT);
 
-	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the Editor Data/Settings Folder."));
+	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the Project Data/Settings Folder."));
```

Comment:
```suggestion
	settings_menu->set_item_tooltip(-1, TTR("Screenshots are stored in the User Data (\"user://\") Folder."));
```
Or similar, the other name doesn't name a specific path (except possibly `.godot`)

Reply:
Thank you, and I'm sorry for any misconceptions, have a good day ^